docs/fundamentals/networking/telemetry/event-counters.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Networking Event Counters in .NET
+description: Learn how to consume .NET networking event counters.
+author: MihaZupan
+ms.author: mizupan
+ms.date: 10/18/2022
+---
+
+# Networking Event Counters in .NET
+
+[EventCounters] are .NET APIs used for lightweight, cross-platform, and near real-time performance metric collection.
+
+Networking components are instrumented to publish basic diagnostics information using EventCounters.
+They include information like the following:
+
+- `System.Net.Http` > `requests-started`
+- `System.Net.Http` > `requests-failed`
+- `System.Net.Http` > `http11-connections-current-total`
+- `System.Net.Security` > `all-tls-sessions-open`
+- `System.Net.Sockets` > `outgoing-connections-established`
+- `System.Net.NameResolution` > `dns-lookups-duration`
+
+> [!TIP]
+> For the full list, see [well-known counters].
+
```

Comment:
Add a note that metrics are recommended in .NET 8 or later? And have a link to the metrics page.